src/etherscan.js

```diff
@@ -32,8 +33,16 @@ const validateResponse = (response) => {
  */
 
 const scrapeABI = async (address) => {
+  const networkID = web3Utils.getEtherNetworkId();
+  const networks = {
+    1: 'api',
+    3: 'ropsten',
+    4: 'rinkeby',
+    42: 'kovan',
```

Comment:
Isn't better to put this config into config file?

Reply:
How do you mean? 
Do you mean as an ENV var?

Reply:
No I meant in `config.js`, just put the object in there.

Reply:
good idea